carp.deployment.core/src/commonMain/kotlin/dk/cachet/carp/deployment/application/DeploymentService.kt

```diff
@@ -77,21 +82,31 @@ interface DeploymentService
      * - a deployment with [studyDeploymentId] does not exist
      * - [masterDeviceRoleName] is not present in the deployment
      * - the [deploymentChecksum] does not match the checksum of the expected deployment. The deployment might be outdated.
-     * @throws IllegalStateException when the deployment cannot be deployed yet.
+     * @throws IllegalStateException when the deployment cannot be deployed yet, or the deployment has stopped.
      */
     suspend fun deploymentSuccessful( studyDeploymentId: UUID, masterDeviceRoleName: String, deploymentChecksum: Int ): StudyDeploymentStatus
```

Comment:
We don't really do command/query separation. Most of the requests always return status after doing the command, so the same comments goes for almost all requests. This is more of a convenience so you don't need two requests to have latest state (or infer it yourself).

Personally not clear to me how prepending with `set` would make that clearer. :/ That said, not entirely against using `set`. I think I didn't since that sounded like something you could set 'multiple times'. It is really more of an event.

Reply:
I'm not religious about this, so maybe just add to the docs what firing this implies